packages/edit-site/src/components/global-styles/ui.js

```diff
@@ -58,6 +59,12 @@ function ContextScreens( { name } ) {
 				<ScreenTypographyElement name={ name } element="link" />
 			</GlobalStylesNavigationScreen>
 
+			<GlobalStylesNavigationScreen
+				path={ parentMenu + '/typography/button' }
+			>
+				<ScreenTypographyElement name={ name } element="button" />
+			</GlobalStylesNavigationScreen>
```

Comment:
How about extracting these declarations to a new file called `elements.js` that would export a single variable, such as below? This way it you could manage it from a single file.

```js
export const ElementsUI = [
    {
        name: 'button',
        editTypography: {
		title: __( 'Buttons' ),
		description: __( 'Manage the fonts and typography used on buttons.' ),
        },
        editColors: {
		title: __( 'Buttons' ),
		description: __( 'Set the default colors used for buttons across the site.' ),
        	component: ScreenButtonColor,
        }
    }
]
```

Reply:
I think we should create a _follow up_ Issue to do just this. @scruffian will be the best person to do this as he has his head in the right code. We shouldn't delay on this as it's inevitable we will want to extend with more controls in future. I propose making the follow up Issue now with clear instructions on the likely path forward and references to the appropriate files.

Reply:
Same as @getdave suggested once we merge this duplication I think it'll become even easier to concoct an abstraction to simplify.